src/Skedify.test.js

```diff
@@ -324,7 +324,7 @@ describe('API', () => {
       expect(await matchRequest(SDK.subjects(1))).toMatchSnapshot()
     })
 
-    it('should be possible to add includes to the subjects collection', async () => {
+    it('should be possible to add includes to the subjects & offices collection(s)', async () => {
```

Comment:
I don't think this change is necessary because otherwise we have to add all resources and all includes and all combinations which becomes a bit messy. (I'll push a commit to remove this again).